src/plugins/intel_cpu/src/nodes/bin_conv.cpp

```diff
@@ -1392,20 +1392,20 @@ void BinaryConvolution::executeReference(const uint8_t* src,
         int32_t a = 0;
         ker(a, g, mb, oc, oh, ow);
 
-        float base_value;
-        if (pad_value == 0.0F) {
-            const int i_left_overflow = nstl::max(0, (padL - ow * KSW));
-            const int i_right_overflow = nstl::max(IW, (ow * KSW + (KW - 1) * (KDW + 1) - padL + 1)) - IW;
-            const int kw_padding = KW - div_up(i_left_overflow, (KDW + 1)) - div_up(i_right_overflow, (KDW + 1));
+        auto base_value = [&]() -> float {
```

Comment:
Create functor to just initialize variable looks like overhead and variable is always initialized.
It looks like rule not detect real issues in code.